### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by cs-utkarsh-surana**
#### UTCs:  

- [x] Connector installation verified.  
- [x] Connector logo verified.  
- [x] Actions and Playbook list verified.  
- [x] Playbook tags verified.  
- [x] All Playbook are in info mode verified.  
- [x] All Playbook are in inactive mode verified.

### CODE REVIEW

File: middesk/info.json [hunk 1]
```diff
@@ -0,0 +1,1136 @@
...
+        "name": "auth_type",
+        "value": "API keys",
+        "description": "Select the method using which you will get authenticated to access the Middesk APIs. You can choose between API keys or OAuth access tokens - on behalf of account owners. For more information, see the Getting Access Code section.",
+        "onchange": {
+          "API keys": [
+            {
+              "title": "API Key",
+              "name": "token",
+              "type": "text",
+              "description": "Specify the PolySwarm API Key through which you will connect and perform actions on PolySwarm.",
```
Comment (cs-harsh-sharma) [inline middesk/info.json]: In desc. and tooltip Is it polyswarm..?

File: middesk/info.json [hunk 2]
```diff
@@ -0,0 +1,1136 @@
...
+    {
+      "title": "Update Business",
+      "description": "Updates an existing business based on the id of a Business you have provided",
+      "operation": "update_business",
+      "category": "investigation",
+      "annotation": "update_business",
+      "parameters": [
+        {
+          "title": "ID",
+          "description": "Specify the id of the Business to retrieve",
```
Comment (cs-harsh-sharma) [inline middesk/info.json]: is it really retrieve?
or update.

Both tooltip and desc

File: middesk/info.json [hunk 3]
```diff
@@ -0,0 +1,1136 @@
...
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "tags",
+          "tooltip": "Specify the tags. You can specify multiple tags as comma-separated values. Any tag can be added (Note that low_priority is a reserved tag, and will affect business turnaround time)."
+        },
+        {
+          "title": "Formation",
+          "description": "Specify the business formation (Two-Letter abbreviation for the state the business was formed in)",
```
Comment (cs-harsh-sharma) [inline middesk/info.json]: Placeholder or e.g. required

File: middesk/info.json [hunk 4]
```diff
@@ -0,0 +1,1136 @@
...
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "external_id",
+          "tooltip": "Specify the custom field identifier for system outside Middesk"
+        },
+        {
+          "title": "Orders",
+          "description": "Specify the orders for the types of products you would like Middesk to fulfill for the business you're submitting.You can specify multiple orders as comma-separated values.For more details (https://docs.middesk.com/reference/order). Valid values are identity, tin, documents, and liens.",
```
Comment (cs-harsh-sharma) [inline middesk/info.json]: Desc can be better

File: middesk/info.json [hunk 5]
```diff
@@ -0,0 +1,1136 @@
...
+  "label": "Middesk",
+  "version": "1.0.0",
+  "description": "Middesk is an identity platform that automates business verification and underwrites decisions. It also provides data on businesses and notifies service providers of changes to its customer base allowing them to form an accurate picture of their customers and offer the critical products their customers need to establish,operate, and maintain their businesses.",
+  "publisher": "Fortinet",
+  "cs_approved": false,
+  "cs_compatible": true,
+  "category": "IT Service Management",
+  "icon_small_name": "small.png",
+  "icon_large_name": "large.png",
+  "ingestion_supported": true,
```
Comment (cs-harsh-sharma) [inline middesk/info.json]: Is ingestion supported?

Comment (cs-utkarsh-surana) [reply middesk/info.json:12]: NO

File: middesk/info.json (hunk 4 above)
Comment (cs-utkarsh-surana) [reply middesk/info.json]: Done

File: middesk/info.json (hunk 3 above)
Comment (cs-utkarsh-surana) [reply middesk/info.json]: Done

File: middesk/info.json (hunk 2 above)
Comment (cs-utkarsh-surana) [reply middesk/info.json]: Done

File: middesk/info.json (hunk 1 above)
Comment (cs-utkarsh-surana) [reply middesk/info.json]: Done

File: middesk/info.json (hunk 5 above)
Comment (cs-utkarsh-surana) [reply middesk/info.json]: Done

File: middesk/info.json [hunk 6]
```diff
@@ -0,0 +1,1136 @@
...
+  "configuration": {
+    "fields": [
+      {
+        "title": "Get Access Token",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "select",
+        "options": [
+          "API keys",
```
Comment (cs-prerna-joshi) [inline middesk/info.json]:
```suggestion
          "API Key",
```

File: middesk/info.json [hunk 7]
```diff
@@ -0,0 +1,1136 @@
...
+          "OAuth access tokens - on behalf of account owners"
+        ],
+        "name": "auth_type",
+        "value": "API keys",
+        "description": "Select the method using which you will get authenticated to access the Middesk APIs. You can choose between API keys or OAuth access tokens - on behalf of account owners. For more information, see the Getting Access Code section.",
+        "onchange": {
+          "API keys": [
```
Comment (cs-prerna-joshi) [inline middesk/info.json]:
```suggestion
          "API Key": [
```

File: middesk/info.json [hunk 8]
```diff
@@ -0,0 +1,1136 @@
...
+    "fields": [
+      {
+        "title": "Get Access Token",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "select",
+        "options": [
+          "API keys",
+          "OAuth access tokens - on behalf of account owners"
```
Comment (cs-prerna-joshi) [inline middesk/info.json]:
```suggestion
          "OAuth Access Token - on behalf of account owners"
```

File: middesk/info.json [hunk 9]
```diff
@@ -0,0 +1,1136 @@
...
+        "type": "checkbox",
+        "name": "verify_ssl",
+        "value": true,
+        "description": "Specifies whether the SSL certificate for the server is to be verified or not. By default, this option is set as True."
+      }
+    ]
+  },
+  "operations": [
+    {
+      "title": "List Businesses",
```
Comment (cs-prerna-joshi) [inline middesk/info.json]:
```suggestion 
      "title": "Get Businesses List",
```

File: middesk/info.json [hunk 10]
```diff
@@ -0,0 +1,1136 @@
...
+    },
+    {
+      "title": "Get Business",
+      "description": "Retrieves the details of an existing business based on the id of a Business you have provided",
+      "operation": "get_business",
+      "category": "investigation",
+      "annotation": "get_business",
+      "parameters": [
+        {
+          "title": "ID",
```
Comment (cs-prerna-joshi) [inline middesk/info.json]:
```suggestion
          "title": "Business ID",
```

File: middesk/info.json [hunk 11]
```diff
@@ -0,0 +1,1136 @@
...
+          "formation": {
+            "formation_state": ""
+          },
+          "names": ""
+        }
+      },
+      "enabled": true
+    }
+  ]
+}
```
Comment (cs-prerna-joshi) [inline middesk/info.json:1136]: please update actions in CRUD.

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**